packages/builder/tests/test_builder.rs

```diff
@@ -23,7 +23,7 @@ struct EnvTwo {
 #[test]
 fn test_env_methods() {
     let mut builder = PolywrapClientConfig::new();
-    let uri = uri!("wrap://ens/wrapper.eth");
+    let uri = uri!("wrap://authority/wrapper");
```

Comment:
Let's use `mock` (or `test`) for the authority instead of `authority` to make the intention more clear to the reader